monkey/monkey_island/cc/services/attack/technique_reports/T1504.py

```diff
@@ -25,14 +24,23 @@ class T1504(AttackTechnique):
     def get_report_data():
         data = {'title': T1504.technique_title(), 'info': []}
 
-        powershell_profile_modification_info = list(mongo.db.telemetry.aggregate(T1504.query))
+        shell_startup_files_modification_info = list(mongo.db.telemetry.aggregate(T1504.query))
+
+        powershell_startup_modification_info = []
+        for shell_startup_file_result in shell_startup_files_modification_info[0]['result']:
+            # only want powershell startup files
+            if "profile.ps1" in shell_startup_file_result[0]:
+                powershell_startup_modification_info.append({
+                    'machine': shell_startup_files_modification_info[0]['machine'],
+                    'result': shell_startup_file_result
+                    })
```

Comment:
Almost the same code as with bash. Better extract it to a function that takes filename ([".bash", ".profile"] or ["profile.ps1"]) and all info.